core/src/main/java/com/radixdlt/RadixNodeModule.java

```diff
@@ -425,6 +428,11 @@ protected void configure() {
         properties.get("api.engine_state.port", DEFAULT_ENGINE_STATE_API_PORT);
     install(new EngineStateApiServerModule(engineStateApiBindAddress, engineStateApiPort));
 
+    final var meshApiBindAddress =
+        properties.get("api.mesh.bind_address", DEFAULT_MESH_API_BIND_ADDRESS);
+    final var meshApiPort = properties.get("api.mesh.port", DEFAULT_MESH_API_PORT);
+    install(new MeshApiServerModule(meshApiBindAddress, meshApiPort));
```

Comment:
I think it would be good to also have an "is enabled" flag for this... (defaulting to false, but overriden to true locally in `default.config`) - to avoid installing/start/running/stopping the mesh server for node runners who don't want/need it.

Reply:
Done